transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java

```diff
@@ -68,7 +70,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         getSupportActionBar().setDisplayShowHomeEnabled(true);
-        getSupportActionBar().setTitle(R.string.title_activity_list_transfer_method);
+        getSupportActionBar().setTitle(R.string.mobileTransferMethodsHeader);
```

Comment:
I see 2 changes in this PR for header `title_activity_list_transfer_method` and `mobileTransferMethodsHeader`. Remove the one that's not used

Reply:
`title_activity_list_transfer_method` this is just for manifest

Reply:
ok looks like duplicating the keys on every other module just works, removing the other keys and just use one single key `mobileTransferMethodsHeader`